### PR TITLE
update tech architecture diagram

### PR DESCRIPTION
**Pull request by fernandoocampo**
* update core and tx exporter integrations

### CODE REVIEW

File: docs/diagram-tech-arq.plantuml [hunk 1]
```diff
@@ -12,22 +12,26 @@ left to right direction
 
 APIGateway(apiGateway, "API Gateway", "External API Patterns")
 ElasticKubernetesService(creditCrytoService, "Credit Crypto Service", "Saga Pattern")
-Blockchain(blockchainService, "Core Service", "Smart Contract")
+ElasticKubernetesService(blockchainService, "Core", "Block chain client")
```
Comment (fernandoocampo) [inline docs/diagram-tech-arq.plantuml:15]: actually this one is a service that communicates with the blockchain, that's the one that knows how to do so

File: docs/diagram-tech-arq.plantuml [hunk 2]
```diff
@@ -12,22 +12,26 @@ left to right direction
...
 ManagedStreamingforApacheKafka(kafka, "Kafka", "Message Style Pattern")
 Database(databaseBlockchain, "Database", "Postgres Database")
-ElasticKubernetesService(erpService, "ERP", "Last mile service")
+Blockchain(blockchain, "blockchain", "blockchain")
```
Comment (fernandoocampo) [inline docs/diagram-tech-arq.plantuml:18]: this is the blockchain that blockchainservice will call in order to add a tx or add a contract. The blockchain can be any: ethereum, btc, ada and so on.

File: docs/diagram-tech-arq.plantuml [hunk 3]
```diff
@@ -12,22 +12,26 @@ left to right direction
...
+ElasticKubernetesService(exportService, "tx exporter", "Last mile service")
```
Comment (fernandoocampo) [inline docs/diagram-tech-arq.plantuml:19]: this service will be in charge of transform tx payloads and send them to an external app or service.

File: docs/diagram-tech-arq.plantuml [hunk 4]
```diff
@@ -12,22 +12,26 @@ left to right direction
...
 ElasticKubernetesService(analyticsService, "Analytics", "Last mile service")
 ElasticKubernetesService(auditService, "Audit", "Last mile service")
 Database(databaseBusinessRequirements, "Database", "Postgres Database")
+component "Eexternal Platform" as externalapp <<service layer>>
```
Comment (fernandoocampo) [inline docs/diagram-tech-arq.plantuml:23]: this is the external platform or service that will receive notification of the transactions we process on credit crypto app.

**Comment by wilorios** [comment]
Ok, merged the "feature/tech-arch" with "dev" branch

**Pull request merged**